packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -43,3 +44,125 @@ export function pretty(arg: unknown): number | string {
 	}
 	return JSON.stringify(arg);
 }
+
+const schemaFactory = new SchemaFactory("test");
```

Comment:
Nit: these new utilities are fairly benchmark-specific. It might make sense to either put them directly in `simpleTree.bench.ts` or make a separate benchmark utilities module if you think other test suites would want to use these.

Reply:
Moved them to benchmarkUtilities.ts